app/src/main/java/nl/tudelft/bbw/crawler/BlockWriter.java

```diff
@@ -0,0 +1,43 @@
+package nl.tudelft.bbw.crawler;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.Writer;
+import java.util.Map;
+
+/**
+ * Class to write a block to a file
+ */
+final class BlockWriter {
+
+    /**
+     * Class attributes
+     */
+    public static final String OUTPUT_PATH = "output.json";
```

Comment:
Why does the BlockWriter write to a default path instead of taking a path as argument?